docs/Branches.md

```diff
@@ -56,7 +56,13 @@ You can use any of the branch names as the argument to `--scheme`, such as `mast
 
 - Swift: new commits go to `master`
 
-- LLVM Project: new commits go to `swift/master-next`
+- LLVM Project (Non-lldb commits): New commits go to `master` in the upstream [llvm-project](https://github.com/llvm/llvm-project).
+
+  ... then these commits can be cherry-picked to an appropriate, `swift/master` aligned `apple/stable/*` branch in Apple's fork of [llvm-project](https://github.com/apple/llvm-project). Please see
+  [Apple's branching scheme](https://github.com/apple/llvm-project/blob/apple/master/apple-docs/AppleBranchingScheme.md)
+  document to determine which `apple/stable/*` branch you should cherry-pick to.
+
+- LLVM Project (LLDB commits): new commits go to `swift/master-next`
```

Comment:
This isn't entirely correct, non-internal, non-swift related commits should still go to `apple/stable/*` branches for lldb. Maybe we should say something like 

> Changes that depend on Swift (this only applies to LLDB) should go on `swift/master-next`.